### PR TITLE
HTML: test fieldset rendering with vertical writing-mode

### PR DESCRIPTION
**Pull request by zcorpan**
Bugs:
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/7583968/
https://bugs.chromium.org/p/chromium/issues/detail?id=476234

### CODE REVIEW

**Review by mstensho — APPROVED** [review]
Nits and suggestions. Nothing that blocks landing, though.

File: html/rendering/non-replaced-elements/the-fieldset-element-0/resources/fieldset-vertical.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+body > div { display: inline-block }
+fieldset, .f { padding: 0; height:10em; width:2em; border:1em groove; margin: 0em; line-height:1 }
+legend, .l { padding: 0; width: 1em }
+.l {
+  background: white; /* overlap the border to emulate the border not being painted */
+  display: table; /* shrink-wrap */
```
[the remark this answers is not in the document]
Comment (zcorpan) [reply html/rendering/non-replaced-elements/the-fieldset-element-0/resources/fieldset-vertical.css]: Yep, but it's not supported in Firefox (only prefixed).

[the remark this answers is not in the document]
Comment (mstensho) [reply html/rendering/non-replaced-elements/the-fieldset-element-0/resources/fieldset-vertical.css]: Ok, table it is. :)

**Review by mstensho — APPROVED** [review]

File: html/rendering/non-replaced-elements/the-fieldset-element-0/resources/fieldset-vertical.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (mstensho) [reply html/rendering/non-replaced-elements/the-fieldset-element-0/resources/fieldset-vertical.css]: That causes trouble in Blink, on the other hand. Tables don't support subpixels, so there are minor rendering differences on Mac.

**Pull request merged**